internal/integration/tidb_test.go

```diff
@@ -0,0 +1,1074 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+package integration
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+	"testing"
+
+	"ariga.io/atlas/sql/mysql"
+	"ariga.io/atlas/sql/schema"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/stretchr/testify/require"
+)
+
+func tidbRun(t *testing.T, fn func(*myTest)) {
+	myTests.Do(func() {
+		myTests.drivers = make(map[string]*myTest)
+		for version, port := range map[string]int{"TiDB": 4309} {
```

Comment:
Only one? You have 2 in the docker-compose. 

If it's only one remove the loop.